easybuild/easyconfigs/p/Pindel/Pindel-0.2.5b9-20170508-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,57 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# Author: Jonas Demeulemeester
+# The Francis Crick Insitute, London, UK
+##
+
+easyblock = 'CmdCp'
+
+name = 'Pindel'
+version = '0.2.5b9-20170508'
+
+homepage = 'http://gmt.genome.wustl.edu/packages/pindel/'
+description = """ Pindel can detect breakpoints of large deletions, medium sized
+ insertions, inversions, tandem duplications and other structural variants at single-based
+ resolution from next-gen sequence data. It uses a pattern growth approach to identify the
+ breakpoints of these variants from paired-end short reads. """
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+toolchainopts = {'openmp': True}
+
+# no official release provided for version 0.2.5b9 (20160729),
+# therefore, pull last commit (20170508) to this version from master
```

Comment:
@jdemeul This comment isn't accurate anymore? And the relevant comment was moved out of context, probably an artifact of using `--inject-checksums`?

Reply:
@boegel It's accurate, the version currently in the repo is 0.2.5b9 - 20160729 which lacks an official release. There have however been a few additional bug fixes committed since then so I'm pulling the most recent commit (dated 20170508) and indicating this in the version number.

I'll move the comment up to where the version is specified.